includes/lib/usage-tracking/class-usage-tracking-base.php

```diff
@@ -213,6 +207,32 @@ public function send_event( $event, $properties = array(), $event_timestamp = nu
 		// to ever add event tracking at the user level.
 		$properties['_ui'] = str_replace( 'www.', '', wp_parse_url( site_url(), PHP_URL_HOST ) );
 		$properties['_ul'] = $user->user_login;
+
+		return $this->send_anonymous_event( $event, $properties, $event_timestamp );
+
+	}
+
+
+	/**
+	 * Send an event to Tracks.
+	 *
+	 * @param string   $event           The event name. The prefix string will be
+	 *                                  automatically prepended to this, so please supply this string without a
+	 *                                  prefix.
+	 * @param array    $properties      Event Properties.
+	 * @param null|int $event_timestamp When the event occurred.
+	 *
+	 * @return bool
+	 **/
```

Comment:
I see that the other functions in this file close the PHPDoc with `**/`, but considering it's a new function, maybe we could add it correctly `*/`. WDYT?
Or we could also take the opportunity to replace all of them in this file! =)

Reply:
Cleaned them up in a54b8b9